libs/docs/src/docs/2.1.0/quickstart-sandbox.md

```diff
@@ -0,0 +1,374 @@
+---
```

Comment:
Unfortunately this procedure is based on the [Dashboard Devfile editor that has been disabled in Eclipse Che 7.65 and that will be disabled starting from next version of OpenShift Dev Spaces](https://github.com/eclipse/che/issues/22115) (the rational behind this can be found [in this epic](https://github.com/eclipse/che/issues/21606)). I would remove this 2.1.0 section to avoid confusions.